commit d1a89882d368b05748c828112cf01c05a44e32f4
Author: Andy Postnikov <apostnikov@gmail.com>
Date:   Fri Sep 30 07:52:50 2022 +0200

    Fix test when session extesion is shared
    
    When extension is build as shared then testing fpm configuration gets no extensions from EXTENSIONS section

diff --git a/sapi/fpm/tests/bug77780-header-sent-error.phpt b/sapi/fpm/tests/bug77780-header-sent-error.phpt
index b5e7691854..89780879a2 100644
--- a/sapi/fpm/tests/bug77780-header-sent-error.phpt
+++ b/sapi/fpm/tests/bug77780-header-sent-error.phpt
@@ -24,7 +24,7 @@
 EOT;
 
 $tester = new FPM\Tester($cfg, $code);
-$tester->start();
+$tester->start(['-dextension=session']);
 $tester->expectLogStartNotices();
 $tester
     ->request(
diff --git a/sapi/fpm/tests/tester.inc b/sapi/fpm/tests/tester.inc
index 1d27ffc2c8..2fb8fbd5fa 100644
--- a/sapi/fpm/tests/tester.inc
+++ b/sapi/fpm/tests/tester.inc
@@ -392,7 +392,7 @@ class Tester
         $configFile = $this->createConfig();
         $desc       = $this->outDesc ? [] : [1 => array('pipe', 'w'), 2 => array('redirect', 1)];
 
-        $cmd = [self::findExecutable(), '-F', '-y', $configFile];
+        $cmd = [self::findExecutable(), '-F', '-y', $configFile, '-dextension_dir=modules'];
 
         if ($forceStderr) {
             $cmd[] = '-O';
